.travis.yml

```diff
@@ -4,5 +4,12 @@ jdk:
 before_install:
   - "echo $JAVA_OPTS"
   - "export JAVA_OPTS=-Xmx512m"
+
+script:
+  - mvn install
+  - mvn test -B
```

Comment:
Calling `mvn install` and `mvn test` is redundant. Probably `mvn install -B` will be adequate.